server/controllers/temporary-accommodation/manage/bookingSearchController.test.ts

```diff
@@ -162,38 +158,11 @@ describe('BookingSearchController', () => {
             tableHeadings: [],
             subNavArr: [],
             crn: searchParameters.crn,
-            errors: {},
             response: paginatedResponse,
             pagination: {},
           })
         },
       )
     })
-
-    describe('when an blank CRN search is submitted', () => {
-      it.each(['provisional', 'confirmed', 'active', 'departed'])(
-        'renders an error for %s bookings',
-        async uiStatus => {
-          const status = (uiStatus === 'active' ? 'arrived' : uiStatus) as BookingSearchApiStatus
-          const searchParameters = bookingSearchParametersFactory.build({ crn: '   ' })
```

Comment:
The previous implementation trimmed the CRN to assert it had a minimum length -- this makes me wonder if we should actually do this for any input for this?

Reply:
I agree. I'm going to trim the users input as part of the rename the search param and completely ignore the param if its empty and treat it as if the user clicked on "Clear"